proxy/src/config.rs

```diff
@@ -587,6 +590,9 @@ pub struct WakeComputeLockOptions {
 impl WakeComputeLockOptions {
     /// Default options for [`crate::console::provider::ApiLocks`].
     pub const DEFAULT_OPTIONS_WAKE_COMPUTE_LOCK: &'static str = "permits=0";
+    /// Default options for [`crate::console::provider::ApiLocks`].
+    pub const DEFAULT_OPTIONS_CONNECT_COMPUTE_LOCK: &'static str =
+        "shards=64,permits=50,epoch=10m,timeout=2s";
```

Comment:
What do you think about making timeout more strict? `2s` is exactly the same timeout as when the connection is establishing.